### PR TITLE
Tested and Fixed Pong with GDScript Demo for 4.0-dev

### PR DESCRIPTION
**Pull request by voylin**
Made the usual changes to the project settings and tested to see if
everything works.

Everything works fine!
Another task which can be checked of #697.

### CODE REVIEW

File: 2d/pong/README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ for game development in Godot, including
 
 Language: GDScript
 
-Renderer: GLES 2
+Renderer: Vulkan
```
Comment (aaronfranke) [inline 2d/pong/README.md:9]: If we're updating this text, we should be specifying if it's Vulkan Clustered or Vulkan Mobile.

Additionally, we should be setting demos to Vulkan Mobile if it doesn't look any different compared to Vulkan Clustered. Especially for GUI-only demos, they will look the same in Vulkan Mobile. Go to Project -> Project Settings -> General tab -> Rendering section -> Vulkan -> Back End -> Select Forward Mobile.

File: 2d/pong/project.godot [hunk 2]
```diff
@@ -15,19 +15,14 @@ config/description="A simple Pong game. This demo shows best practices
...
 run/main_scene="pong.tscn"
 config/icon="res://icon.png"
+config/features=PackedStringArray("4.0")
 
 [display]
 
+window/stretch/mode="canvas_items"
+window/stretch/aspect="expand"
 window/size/width=640
 window/size/height=400
```
Comment (aaronfranke) [inline 2d/pong/project.godot:25]:
```suggestion
window/size/viewport_width=640
window/size/viewport_height=400
```

**Review by aaronfranke — COMMENTED** [review]
Most of this looks good.

File: 2d/pong/project.godot (hunk 2 above)
Comment (voylin) [reply 2d/pong/project.godot]: Thanks! This was something I missed in the settings, changed the settings, squashed and pushed.

**Review by aaronfranke — APPROVED** [review]
Thanks!

**Pull request merged**